.github/workflows/api-build.yml

```diff
@@ -20,11 +20,16 @@ jobs:
     if: github.repository_owner == 'bcgov'
     runs-on: ubuntu-latest
     steps:
+      - name: Install Tekton CLI
```

Comment:
This should be either removed, or updated, as this will always cause this to fail.
I think we need to revert it back to building gdx-agreements-tracker-api-run